deploy/011-upgrade-comptroller.ts

```diff
@@ -0,0 +1,33 @@
+import { ethers } from "hardhat";
+import { DeployFunction, DeployResult, Deployment } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
```

Comment:
I would try to skip it for the main net

Reply:
added a skip function to skip when `network.name !== "bsctestnet"`